.github/workflows/check-sorbet-typing-mode.yml

```
@@ -0,0 +1,68 @@
+name: Sorbet
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+    paths:
+      - '*/lib/dependabot/**/*.rb'  # Trigger the workflow only for Ruby files
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [main]  # Adjust this to the branch you want for push events
+    paths:
+      - '*/lib/dependabot/**/*.rb'  # Trigger the workflow only for Ruby files
+
+permissions:
+  contents: read
+
+jobs:
+  check-sorbet-typing-mode:
+    name: Sorbet Typing Mode Check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

Personally, I've started omitting the `name` field for clearly self-describing steps... otherwise it's like code comments that repeat what the code does... that makes sense for a complicated step, like "Verify Sorbet Typing Mode"... 

`actions/checkout` is up to `v4` btw

```suggestion
      - uses: actions/checkout@v4
```